src/goose.rs

```diff
@@ -2444,6 +2582,7 @@ impl Hash for GooseTask {
 }
 
 #[cfg(test)]
+#[allow(clippy::blacklisted_name)]
```

Comment:
Instead of allowing blacklisted variable names, let's choose more descriptive variables below.